plugins/test-plugin/test/run-test.sh

```diff
@@ -32,17 +32,17 @@ cp nodekey02 02/geth/nodekey
 
 echo -n "supersecretpassword" > passwordfile
 
-$GETH init --datadir=./00 genesis.json
-$GETH init --datadir=./01 genesis.json
-$GETH init --datadir=./02 genesis.json
+# $GETH init --datadir=./00 genesis.json
+# $GETH init --datadir=./01 genesis.json
+# $GETH init --datadir=./02 genesis.json
 
 # miner node
-$GETH --cache.preimages --config config00.toml --authrpc.port 8552 --port 64480 --verbosity=0 --nodiscover --networkid=6448 --datadir=./00/ --mine --miner.etherbase f2c207111cb6ef761e439e56b25c7c99ac026a01 --unlock f2c207111cb6ef761e439e56b25c7c99ac026a01 --http --http.api eth,debug,net --http.port 9545 --password passwordfile --allow-insecure-unlock &
+$GETH --cache.preimages --config config00.toml --authrpc.port 8552 --port 64480 --verbosity=3 --nodiscover --networkid=6448 --datadir=./00/ --mine --miner.etherbase f2c207111cb6ef761e439e56b25c7c99ac026a01 --unlock f2c207111cb6ef761e439e56b25c7c99ac026a01 --http --http.api eth,debug,net --http.port 9545 --password passwordfile --allow-insecure-unlock &
```

Comment:
I would prefer is we kept the verbosity at 0 for the miner. I realize its a trivial fix but the test is chatty enough as it is.